### ISSUE TITLE
Adobe comments on WCAG 3.0 FPWD

### ISSUE THREAD

**Issue by awkawk** [w3c/silver#398]
Thank you for the opportunity to comment on WCAG 3.0. Adobe supports the efforts to develop a major version update of WCAG and would like to offer comments that are minor in scope (some editorial) and some which are more significant in scope. We are providing a group of minor comments below and multiple separate comments on major areas as separate GitHub issues:

Minor: 
1.	The Abstract  says that 3.0 incorporates UAAG and ATAG, but not WCAG “WCAG 3.0 will incorporate content from and partially extend User Agent Accessibility Guidelines 2.0 [UAAG20] and Authoring Tool Accessibility Guidelines 2.0 [ATAG20].” This may be implied but we believe that it should be explicit in order to reassure people that there is some relationship beyond succession.
2.	“Following these guidelines will make content more accessible to people with a wide range of disabilities, including accommodations for blindness, low vision and other vision impairments; deafness and hearing loss; limited movement and dexterity; speech disabilities; sensory disorders; cognitive and learning disabilities; and combinations of these.” – More accessible than what? 
3.	Why change the URI pattern? /TR/WCAG21 vs /TR/WCAG-3.0 This is more difficult to type and changes a familiar pattern.
4.	“However, changing technology and changing needs of people with disabilities have led to the need for a new model to address content accessibility more comprehensively and flexibly.” – do we believe that the needs of people with disabilities have changed? If so, this should be clarified with more specifics. 
5.	In the Plain Language summary for “additional documentation and scoring information”: Functional needs: how to solve the access problems that people face. – this description is a verb but the term is a noun. Seems like it should be “access problems that people face” 
6.	Figure 3 is hard to follow as it doesn’t seem to have a clear starting point. This is somewhat more consequential as it impacts the ability of people to understand the specification, which is a current problem among reviewers in our organization.

**Comment by bruce-usab** [comment]
Just my two bits on @awkawk last item (6) since it happens that today I was looking at [Figure 3](https://www.w3.org/TR/2021/WD-wcag-3.0-20210121/#additional-documentation-and-scoring-information) for what feels like the first time.  I did a view source to look for <q>a text alternative that serves the equivalent purpose</q> since the paragraph before the image was not really doing it for me.
- id is https://www.w3.org/TR/2021/WD-wcag-3.0-20210121/#guidelines-full-structure-diagram
- aria-labelledby is ttps://www.w3.org/TR/2021/WD-wcag-3.0-20210121/#guidelines--full-structure-description

**Comment by jpascalides** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
